core/src/test/java/io/opencensus/stats/MeasureMapTest.java

```diff
@@ -77,87 +66,63 @@ public void testBuilder() {
-    MeasureMap metrics6 = MeasureMap.builder().put(M2, 2.0).put(M3, 200L).build();
-    assertEquals(MeasureMap.builder().put(M3, 100L).put(M1, 1.0).put(M3, 300L).build().iterator(),
-        metrics5.iterator());
-    assertEquals(MeasureMap.builder().put(M2, 2.0).put(M3, 200L).put(M2, 3.0).build().iterator(),
-        metrics6.iterator());
+    assertEquals(MeasureMap.builder().put(M3, 100L).put(M1, 1.0).put(M3, 300L).build(),
+        measurement3, measurement1);
+    assertEquals(MeasureMap.builder().put(M2, 2.0).put(M3, 100L).put(M2, 3.0).build(),
+        measurement2, measurement3);
   }
 
   private static final DoubleMeasure M1 = makeSimpleDoubleMeasure("m1");
   private static final DoubleMeasure M2 = makeSimpleDoubleMeasure("m2");
   private static final LongMeasure M3 = makeSimpleLongMeasure("m3");
   private static final LongMeasure M4 = makeSimpleLongMeasure("m4");
 
-  private static final DoubleMeasure makeSimpleDoubleMeasure(String measure) {
+  private static final Measurement measurement1 = Measurement.DoubleMeasurement.create(M1, 1.0);
+  private static final Measurement measurement2 = Measurement.DoubleMeasurement.create(M2, 2.0);
+  private static final Measurement measurement3 = Measurement.LongMeasurement.create(M3, 100L);
+  private static final Measurement measurement4 = Measurement.LongMeasurement.create(M4, 200L);
+
+  private static DoubleMeasure makeSimpleDoubleMeasure(String measure) {
     return Measure.DoubleMeasure.create(
         measure, measure + " description", "1");
   }
 
-  private static final LongMeasure makeSimpleLongMeasure(String measure) {
+  private static LongMeasure makeSimpleLongMeasure(String measure) {
     return Measure.LongMeasure.create(
         measure, measure + " description", "1");
   }
 
-  private static void assertEquals(
-      Iterator<Measurement> expected, Iterator<Measurement> actual) {
-    while (expected.hasNext() && actual.hasNext()) {
-      Measurement expectedMeasurement = expected.next();
-      Measurement actualMeasurement = actual.next();
-      assertThat(expectedMeasurement.getMeasure().getName())
-          .isEqualTo(actualMeasurement.getMeasure().getName());
-      assertThat(expectedMeasurement.match(new GetDoubleValue(), new GetLongValue()))
-          .isEqualTo(actualMeasurement.match(new GetDoubleValue(), new GetLongValue()));
-    }
-    assertThat(expected.hasNext()).isFalse();
-    assertThat(actual.hasNext()).isFalse();
-  }
-
-  private static final class GetDoubleValue implements Function<DoubleMeasurement, Object> {
-
-    @Override
-    public Double apply(DoubleMeasurement arg) {
-      return arg.getValue();
-    }
-  }
-
-  private static final class GetLongValue implements Function<LongMeasurement, Object> {
-
-    @Override
-    public Long apply(LongMeasurement arg) {
-      return arg.getValue();
-    }
+  private static void assertEquals(MeasureMap metrics, Measurement... measurements) {
```

Comment:
I think a name like `assertContains` would be more descriptive now that it isn't comparing for equality.